src/aat/resources/features/F-065/S-160.td.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "must return 401 when request does not provide valid authentication credentials",
+
+  "_guid_": "S-160",
+  "_extends_": "F-065_Test_Data_Base",
+
+  "specs": [
+    "an active profile in CCD",
+    "does not provide valid authentication credentials to the operation",
+    "contains an HTTP-401 Unauthorized"
+  ],
+
+  "request": {
+    "headers": {
+      "Authorization": "abc"
```

Comment:
Can you please rename it so something which signifies "anInvalidAuthorizationString".

Reply:
Updated :       "Authorization": "anInvalidAuthorizationString"